monkey/common/hard_coded_manifests/hard_coded_fingerprinter_manifests.py

```diff
@@ -1,46 +1,54 @@
-from common.agent_plugins import AgentPluginManifest, AgentPluginType
+from common.agent_plugins import AgentPluginManifest, AgentPluginType, PluginName
 from common.operating_system import OperatingSystem
 
 HARD_CODED_FINGERPRINTER_MANIFESTS = {
     "smb": AgentPluginManifest(
-        name="smb",
+        name=PluginName("smb"),
         plugin_type=AgentPluginType.FINGERPRINTER,
         supported_operating_systems=(OperatingSystem.LINUX, OperatingSystem.WINDOWS),
         target_operating_systems=(OperatingSystem.LINUX, OperatingSystem.WINDOWS),
         title="SMB Fingerprinter",
-        version="1.0.0",
+        version="1.0.0",  # type: ignore [arg-type]
```

Comment:
Why do we need to ignore these? Just create `PluginVersion` objects?

Reply:
Tried that, it didn't work.

Reply:
Why? What does it expect?

Reply:
I am not really sure with `PluginVersion`. The AgentPluginManifest has some config that is doing a lambda for the PluginVersion